connecting-to-testnet/index.md

```diff
@@ -4,14 +4,29 @@ parent:
 order: 0
 ---
 
-# Cosmos Hub Testnet
+# What is a testnet?
 
-The [Cosmos Network](https://cosmos.network) is an expanding ecosystem of connected blockchains. Many blockchains in the ecosystem are built with [Tendermint Core](https://docs.tendermint.com/), [Cosmos SDK](https://docs.cosmos.network/) and connected with [IBC](https://ibcprotocol.org/). One of these blockchains is the [Cosmos Hub](https://cosmos.network/features). Mainnet of the Cosmos Hub has been online since early 2019 and you can learn how to [join the network as a validator](https://hub.cosmos.network). In this guide you'll learn how to start a full-node for and interface with a Cosmos Hub **testnet**.
+A testnet in our purpose describes a network create with the Cosmos SDK that has been created to test applications.
+It is created so app creator can connect and use the functionality they expect with a real network.
+The state of a testnet can be reset at any time.
+
+
+# The Cosmos Hub Testnet
+
+In this tutorial, we will connect to the Cosmos testnetwork. We will learn how to:
+
+- Get and install the software
+- Start the Node
+- Use the Cosmos SDK CLI
+- Connect to the public endpoints
+
+The [Cosmos Network](https://cosmos.network) is an expanding ecosystem of connected blockchains. Many blockchains in the ecosystem are built with [Tendermint Core](https://docs.tendermint.com/), [Cosmos SDK](https://docs.cosmos.network/) and connected with [IBC](https://ibcprotocol.org/). One of these blockchains is the [Cosmos Hub](https://cosmos.network/features). Mainnet of the Cosmos Hub has been online since early 2019 and you can learn how to [join the network as a validator](https://hub.cosmos.network/main/validators/validator-setup.html). In this guide you'll learn how to start a full-node for and interface with a Cosmos Hub **testnet**.
```

Comment:
I still think that it's better to have a single `h1`, move `The Cosmos Network is an expanding ecosystem...` paragraph to the beginning and follow it with "What is a testnet..." paragraph. @barriebyron wdyt?